classes/institution/DAO.inc.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+/**
+ * @file classes/institution/DAO.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class DAO
+ * @ingroup institution
+ *
+ * @see Institution
+ *
+ * @brief Operations for retrieving and modifying Institution objects.
+ */
+
+namespace PKP\institution;
+
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\LazyCollection;
+use PKP\core\EntityDAO;
+use PKP\core\SoftDeleteTrait;
+use PKP\services\PKPSchemaService;
+
+class DAO extends EntityDAO
+{
+    use SoftDeleteTrait;
+
+    /** @copydoc EntityDAO::$schema */
+    public $schema = PKPSchemaService::SCHEMA_INSTITUTION;
+
+    /** @copydoc EntityDAO::$table */
+    public $table = 'institutions';
+
+    /** @copydoc EntityDAO::$settingsTable */
+    public $settingsTable = 'institution_settings';
+
+    /** @copydoc EntityDAO::$primarykeyColumn */
+    public $primaryKeyColumn = 'institution_id';
+
+    /** @copydoc EntityDAO::$primaryTableColumns */
+    public $primaryTableColumns = [
+        'id' => 'institution_id',
+        'contextId' => 'context_id',
+        'ror' => 'ror',
+        'deletedAt' => 'deleted_at'
+    ];
+
+    /**
+     * Instantiate a new DataObject
+     */
+    public function newDataObject(): Institution
+    {
+        return App::make(Institution::class);
+    }
+
+    /**
+     * Check if an institution exists with this ID and context ID
+     */
+    public function existsInContext(int $id, int $contextId): bool
```

Comment:
(Again just a preference, but rather than `$id` I'd suggest using `$institutionId`. That way it's clear at a glance what kind of ID we're talking about.)